CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 2.5.0
+
+* **Experimental: [Playwright](/playwright) helper introduced**.
+
+> [Playwright](https://github.com/microsoft/playwright/) is an alternative to Puppeteer which works very similarly to it but adds cross-browser support with Firefox and Webkit. Until v1.0 Playwright API is not stable but we introduce it to CodeceptJS so you could try it.
```

Comment:
is an alternative to Puppeteer which works very similarly to it but adding cross-browser support with Firefox and Webkit that brings Playwright to the next level. Until Playwright v1.0 is released, current APIs are not stable but we'd like to  introduce it to CodeceptJS so you could have chance to try it out.